dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -64,6 +64,9 @@
     E4010( "Attribute `{0}` is not supported for type `{1}`." ),
     E4011( "Attribute `{0}` is required, but no value was found." ),
     E4012( "Attribute `{0}` contains elements of different period type than the data set it was added to" ),
+    E4014( "Invalid Closing date `{0}`, must be after Opening date `{1}`"),
```

Comment:
Can be reused with the code from orgUnit closing/opening?

Reply:
yes, will update this after that PR for orgUnit is merged.